grafana-oncall.yaml

```diff
@@ -0,0 +1,196 @@
+package:
+  name: grafana-oncall
+  version: 1.14.1
+  epoch: 0
+  description: "Developer-friendly incident response with brilliant Slack integration"
+  copyright:
+    - license: GPL-3.0-or-later
+  dependencies:
+    runtime:
+      - bash
+      - git
+      - libffi
+      - libpq
+      - mariadb-connector-c
+      - netcat-openbsd
+      - pcre
+      - python-${{vars.py-version}}
+
+vars:
+  py-version: 3.12
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - git-lfs
+      - libffi-dev
+      - linux-headers
+      - mariadb-connector-c-dev
+      - openssf-compiler-options
```

Comment:
given this is enabled by default, we can remove explicit mention of this package. 
```suggestion

```